docs/source/development/defining_new_models.md

```diff
@@ -300,6 +307,14 @@ The method then uses those parsed arguments to actually call the `__init__` meth
 return an initialised instance of the model using the settings. The `from_config`
 method should raise an `InitialisationError` if the configuration fails.
 
+The `from_config` method should also check if any constants have been provided as part
+of the configuration. If they haven't a default set of parameters is generated. If they
+have been supplied they are used to generate a custom set of parameters. The
+{func}`~virtual_rainforest.core.utils.check_constants` utility function is used to check
+that no constant has been supplied with an incorrect name. At least one parameter class
+should be created, but it's fine to split parameters across more classes if that makes
+for clearer code.
+
```

Comment:
Here again, to be clear, I would stick to 'constants' or 'parameters' (whichever you chose)